tests/utils.go

```diff
@@ -102,30 +102,19 @@ import (
 )
 
 const (
-	BinBash                = "/bin/bash"
-	StartingVMInstance     = "Starting a VirtualMachineInstance"
-	WaitingVMInstanceStart = "Waiting until the VirtualMachineInstance will start"
-	EchoLastReturnValue    = "echo $?\n"
+	BinBash                     = "/bin/bash"
+	StartingVMInstance          = "Starting a VirtualMachineInstance"
+	WaitingVMInstanceStart      = "Waiting until the VirtualMachineInstance will start"
+	EchoLastReturnValue         = "echo $?\n"
+	CustomHostPath              = "custom-host-path"
+	DiskAlpineHostPath          = "disk-alpine-host-path"
+	DiskWindowsSysprep          = "disk-windows-sysprep"
+	DiskCustomHostPath          = "disk-custom-host-path"
+	defaultDiskSize             = "1Gi"
+	MigrationWaitTime           = 240
```

Comment:
This looks like a duplicate from what is inside `libmigration`?

Reply:
Rebase leftover! Thank you!